ext/opcache/zend_accelerator_util_funcs.c

```diff
@@ -650,6 +639,16 @@ zend_op_array* zend_accel_load_script(zend_persistent_script *persistent_script,
 #endif
 		}
 
+		/* Relocate pointers from within arena_mem to CG(arena) mem */
+		if (persistent_script->arena_offsets_size > 0) {
+			size_t *cur = (size_t *)((char *) persistent_script->arena_mem + persistent_script->arena_size);
+			size_t *end = (size_t *)((char *) cur + persistent_script->arena_offsets_size);
+			do {
+				void **ptr = (void **) (ZCG(arena_mem) + *cur);
+				*ptr = ARENA_REALLOC(*ptr);
+			} while (++cur < end);
+		}
+
```

Comment:
Or possibly increases performance as generic array duplication is much more expensive as relocating up to one pointer per file [At least in my experience, the most functions typically have not more than two statics and the functions who have are often holding state of an important object - so are called anyway].